winpr/libwinpr/sysinfo/cpufeatures/repo.prop

```diff
@@ -0,0 +1,55 @@
+platform/bionic ed9e6a41c92c9552be84ecc126e29b4604eee246
```

Comment:
Is the repo.prop still usable? I mean is it possible to update the code still? If not we should remove it.